app/templates/directives/grid-contents/grid-contents.jade

```diff
@@ -1,4 +1,4 @@
-.row(ng-repeat="(key, value) in vm.rowsGrid")
+.row(ng-repeat="(key, value) in vm.rowsGrid").alto
```

Comment:
por ejemplo, en este caso, es cierto que `.alto` no nos dice mucho. si es la vista de los contenidos en grid, puede ser un estilo `.content-in-grid` y a esa clase le aplicas un alto, un fondo, un borde o lo que sea